Bellerophon/Bellerophon/Sources/BPManager.swift

```diff
@@ -67,60 +89,48 @@ public class BellerophonManager: NSObject {
                 // An error occurred
                 completionHandler(result: .Failed)
             }
-        })
+        }
     }
 
     // MARK: Private Methods
-    override init() {
-        super.init()
-        NSNotificationCenter.defaultCenter().addObserver(self, selector: Selector(stopTimer()), name: UIApplicationDidEnterBackgroundNotification, object: nil)
-    }
-
-    deinit {
-        NSNotificationCenter.defaultCenter().removeObserver(self)
-    }
-
-    func handleAppStatus(status: BellerophonStatusProtocol) {
+    private func handleAppStatus(status: BellerophonStatusProtocol) {
         if status.apiInactive() {
-            self.displayKillSwitch(status.userMessage())
```

Comment:
`BPManager` treats the passed in view as a black box. It only adds it to its window and displays the window. Even if it defined a default localized message, there would be no way for it to access the view's label to display the message in. That's why the application has to retrieve the view in the  delegate method and set the message manually.